pkg/sitewise/client/client.go

```diff
@@ -144,14 +144,6 @@ func (c *sitewiseClient) BatchGetAssetPropertyAggregatesPageAggregation(ctx cont
 					success = append(success, successEntry)
 				}
 			}
-
-			for i, entry := range success {
-				for _, successEntry := range output.SuccessEntries {
-					if *successEntry.EntryId == *entry.EntryId {
-						success[i].AggregatedValues = append(success[i].AggregatedValues, successEntry.AggregatedValues...)
-					}
-				}
-			}
```

Comment:
This was a bad copy/paste, or rebase or something where this got merged. It re-appended the results that were already appended.